src/main/java/mods/railcraft/common/carts/CartBaseMaintenance.java

```diff
@@ -82,8 +81,8 @@ public String getName() {
     }
 
     public enum CartModeButtonState implements IMultiButtonState {
-        SERVICE(LocalizationPlugin.translate("gui.railcraft.cart.maintenance.mode.service")),
-        TRANSPORT(LocalizationPlugin.translate("gui.railcraft.cart.maintenance.mode.transport"));
+        SERVICE("Service mode"),
+        TRANSPORT("Transport mode");
```

Comment:
Time to remove this `label` field and param

Reply:
Done